flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/type/FlinkReturnTypes.java

```diff
@@ -104,9 +104,9 @@ public RelDataType inferReturnType(SqlOperatorBinding opBinding) {
                 }
             };
 
-    /** Type-inference strategy that always returns "VARCHAR(2000)" with nulls always allowed. */
-    public static final SqlReturnTypeInference VARCHAR_2000_NULLABLE =
-            ReturnTypes.cascade(ReturnTypes.VARCHAR_2000, SqlTypeTransforms.FORCE_NULLABLE);
+    public static final SqlReturnTypeInference VARCHAR_NULLABLE =
```

Comment:
Perhaps name it `VARCHAR_FORCE_NULLABLE` to make more explicit the nullability behaviour?

Reply:
@slinkydeveloper , the `VARCHAR_FORCE_NULLABLE` makes sense to me.